src/components/FilterTop.js

```diff
@@ -104,6 +105,7 @@ function Filter(props) {
     filter.stakeholderUUID || ALL_ORGANIZATIONS
   );
   const [tokenId, setTokenId] = useState(filter?.tokenId || filterOptionAll);
+  const { isLoading } = useContext(SpeciesContext);
```

Comment:
Again, just use `speciesContext.isLoading`

Reply:
acknowledged and updated